projects/demo/src/modules/services/breakpoint/breakpoint.component.ts

```diff
@@ -8,7 +8,6 @@ import {TuiDocExample} from '@taiga-ui/addon-doc';
     changeDetection,
 })
 export class ExampleTuiBreakpointComponent {
```

Comment:
<img width="340" alt="image" src="https://user-images.githubusercontent.com/12021443/229116799-d1e6b4bd-3518-4034-9571-225e6bac691f.png">

Do we need to rename the files?